examples/vision/mixUp.py

```diff
@@ -47,9 +50,9 @@
 tfd = tfp.distributions
 
 """
-## Prepare dataset
+## Prepare the dataset
 
-We will be using the FashionMNIST dataset for this toy example. But this same recipe can
+In this example, we will be using the FashionMNIST dataset. But this same recipe can
```

Comment:
Suggestion:

- Provide a link to FashionMNIST for more info


```suggestion
In this example, we will be using the [FashionMNIST](https://research.zalando.com/welcome/mission/research-projects/fashion-mnist/) dataset. But this same recipe can
```